deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -621,8 +621,7 @@ export class StackComposer {
             })
             // To enable the Migration Console to make requests to other service endpoints with services,
             // it must be deployed after any connected services
-            this.addDependentStacks(migrationConsoleStack, [captureProxyESStack, captureProxyStack, elasticsearchStack,
-                openSearchStack, osContainerStack, migrationStack, kafkaBrokerStack])
+            this.addDependentStacks(migrationConsoleStack, [migrationStack])
```

Comment:
This seems like the correct list, I think we accidentally inverted the dependencies when this was first added.

Reply:
You can remove the comment above this as well. Originally when we used service connect to talk between ECS services, the Migration Console would need to be deployed after those services to have their service name mapping with service connect. We no longer user Service Connect I believe so that shouldn't block us. 

The other concern I have is loading in environment variables. If we are still loading in environment variables from other stacks that create a source or target (not our import case) say in the opensearch container stack case, we wanted to be sure that endpoint existed and could be passed. This seems like something CDK may be smart enough to detect without manually doing it, but it would be nice to confirm that's the case if we still want to support that use case.